src/screenshots.rst

```diff
@@ -0,0 +1,275 @@
+.. spelling::
+
+ fastlane
+ screengrab
+ Screengrab
+ Gradle
+ gradle
+ androidTest
+ apks
+ sdk
+ 
+
+Automating Form-Widgets Guide
+===================================
+
+This guide gives information on how to automate generation of screenshots for form-widgets guide as well as regenerating form-widgets guide whenever new widgets are added.
+
+.. _screenshot-screengrab:
+
+Screenshot Generation using Screengrab
+----------------------------------------
+
+`Screengrab <https://docs.fastlane.tools/actions/screengrab/>`_ is a command line tool, provided by fastlane. fastlane is an open source platform, which provides a toolchain and automates tasks like generating screenshots, dealing with code signing, and releasing your application. Screengrab can be configured to load up different locales, run all the UI tests and take screenshots during the execution of the test on various devices. 
+
+.. _setting-screengrab:
+
+Setting up Screengrab
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To configure screengrab, follow the steps given below:
+
+.. _install-screen:
+
+Installation
+""""""""""""""""""
+
+In the command line type, the following command to check if RubyGem is installed or not. RubyGems is a package manager for the Ruby programming language that provides a standard format for distributing Ruby programs and libraries (in a self-contained format called a *gem*). If Ruby is installed, the command given below would return the version number.
+
+.. code-block:: console
+
+  $ gem -v
+
+If it is not installed, you can install it using the instructions `given here <https://rubygems.org/pages/download>`_. Windows users are recommended to use `RubyInstallers <https://rubyinstaller.org/>`_.
+
+After installing RubyGem install the gems:
+
+Windows and Mac users can use the following commands:
+
+.. code-block:: console
+
+  $ gem install fastlane
```

Comment:
On Mac, you'll need to do a `xcode-select --install` first. Then `sudo` gem install fastlane`.